cmd/run/socket/proxy.go

```diff
@@ -330,7 +330,12 @@ func (p *proxy) proxyHTTP1(cli, srv *bufConn) error {
 			}
 
 			event := p.tmpl.Copy()
-			event.Set("event_id", uuid.New().String())
+			eventID := uuid.New()
+			event.Set("event_id", eventID.String())
+
+			if err := p.global.Logger.AddReader(eventID); err != nil {
```

Comment:
will panic with nil pointer deref if I set `-tracelogs=false` because `p.global.Logger` will be nil and `AddReader` accesses `mu`